src/material/core/tokens/m2/mat/_switch.scss

```diff
@@ -0,0 +1,53 @@
+@use '../../token-utils';
+@use '../../../style/sass-utils';
+
+// The prefix used to generate the fully qualified name for tokens in this file.
+$prefix: (mat, switch);
+
+// Tokens that can't be configured through Angular Material's current theming API,
+// but may be in a future version of the theming API.
+@function get-unthemable-tokens() {
+  @return (
+    unselected-handle-width: 20px,
+    unselected-handle-height: 20px,
+    selected-handle-width: 20px,
+    selected-handle-height: 20px,
+    pressed-handle-width: 20px,
+    pressed-handle-height: 20px,
+    with-icon-handle-width: 20px,
+    with-icon-handle-height: 20px,
+
+    selected-handle-margin-left: 0,
```

Comment:
We should be using start/end instead of left/right since the latter are reversed in RTL.

Reply:
Good catch!